70_Pushover.pm

```diff
@@ -1228,19 +1258,18 @@ sub Pushover_SetMessage2 ($$$$) {
 
         if (
             !$callback
-            || (   $values{action} !~ /^http[s]?:\/\/.*$/
-                && $values{action} =~ /^[\w-]+:\/\/.*$/ )
+            || $values{action} =~ /^[\w-]+:\/\/.*$/
```

Comment:
1) URLs are pasted directly into the Pushover message, so the redirect through the FHEM callback is not required any more

Reply:
This would leak internal only URLs in case FHEM is not reachable via a public link. Therefore, this must be kept to ensure data privacy.

Reply:
Ok then, let's just leave out this change